relayer/relays/beacon/header/syncer/syncer.go

```diff
@@ -207,10 +207,10 @@ func (s *Syncer) GetBlockRoots(slot uint64) (scale.BlockRootProof, error) {
 
 	if s.activeSpec == config.Minimal {
 		blockRootsContainer = &state.BlockRootsContainerMinimal{}
-		beaconState = &state.BeaconStateBellatrixMinimal{}
+		beaconState = &state.BeaconStateCapellaMinimal{}
```

Comment:
I wonder if we shouldn't check the fork version here and use Bellatrix or Capella based on the fork version? Even if it's just to see how we will handle fork versions in production where we will need to support at least 2 hardfork versions at once (before upgrade and after upgrade version).

Reply:
@claravanstaden Yes, absolute nessessary!   In recent commit I use `ForkEpoch` to check IMO maybe more straightforward  and with that we can even do some fork test with our local setup. I would assume same as use `ForkVersion` but correct me if wrong.

https://github.com/Snowfork/snowbridge/blob/38abe484f88fc11a2c64e653f23772b33a052791/relayer/relays/beacon/header/syncer/syncer.go#L219-L230

Reply:
Ah, I realized this may be a little confusing. The `spec` actually refers to the environment variables used in the network. So all the testnets (Goerli, Sepolia) and mainnet will use the `mainnet` spec. The minimal spec is only used for the local network with different environment variables like `SLOTS_PER_EPOCH`. So a Goerli specific spec might not be needed. 

I wonder if we shouldn't lookup the fork using `/eth/v1/beacon/states/{state_id}/fork`? Otherwise I guess we can hardcode the fork epoch.